crates/websocket/src/send.rs

```diff
@@ -223,71 +224,98 @@ pub async fn send_local_notifs(
     }
   }
 
-  // Send notifs to the parent commenter / poster
-  match comment.parent_id {
-    Some(parent_id) => {
-      let parent_comment =
-        blocking(context.pool(), move |conn| Comment::read(conn, parent_id)).await?;
-      if let Ok(parent_comment) = parent_comment {
-        // Get the parent commenter local_user
-        let parent_creator_id = parent_comment.creator_id;
-
-        // Only add to recipients if that person isn't blocked
-        let creator_blocked = check_person_block(person.id, parent_creator_id, context.pool())
-          .await
-          .is_err();
-
-        // Don't send a notif to yourself
-        if parent_comment.creator_id != person.id && !creator_blocked {
-          let user_view = blocking(context.pool(), move |conn| {
-            LocalUserView::read_person(conn, parent_creator_id)
-          })
-          .await?;
-          if let Ok(parent_user_view) = user_view {
-            recipient_ids.push(parent_user_view.local_user.id);
-
-            if do_send_email {
-              let lang = get_user_lang(&parent_user_view);
-              send_email_to_user(
-                &parent_user_view,
-                &lang.notification_comment_reply_subject(&person.name),
-                &lang.notification_comment_reply_body(&comment.content, &inbox_link, &person.name),
-                context.settings(),
-              )
-            }
-          }
-        }
-      }
-    }
-    // Its a post
-    // Don't send a notif to yourself
-    None => {
+  // Send comment_reply to the parent commenter / poster
+  if let Some(parent_comment_id) = comment.parent_comment_id() {
+    let parent_comment = blocking(context.pool(), move |conn| {
+      Comment::read(conn, parent_comment_id)
+    })
+    .await?;
+    if let Ok(parent_comment) = parent_comment {
```

Comment:
Why would this fail, and why silently ignore the failure?

Reply:
I can change